pkg/kubelet/kubelet.go

```diff
@@ -1416,6 +1428,11 @@ func (kl *Kubelet) syncPod(pod *api.BoundPod, containersInPod dockertools.Docker
 		kl.pullImageAndRunContainer(pod, &pod.Spec.Containers[container], &podVolumes, podInfraContainerID)
 	}
 
+	// Create a mirror pod for the static pod if it doesn't exist yet.
+	if !hasMirrorPod && isStaticPod(pod) {
+		kl.mirrorManager.CreateMirrorPod(pod, kl.hostname)
```

Comment:
Lets log the failure here and remove the logging from CreateMirrorPod() that way we only need to do it once.